WalletWasabi.Fluent/ViewModels/Wallets/Send/LabelSelectionViewModel.cs

```diff
@@ -51,6 +51,11 @@ private bool IsPocketEnough(params Pocket[] pockets)
 		var coins = Pocket.Merge(pockets).Coins;
 		var allCoins = Pocket.Merge(_allPockets).Coins;
 
+		if (!coins.Any())
+		{
+			return false;
+		}
```

Comment:
Is this a bug fix? Or related to the PR?

Reply:
Sanity check only. I can remove it.

Reply:
Rather move the check into `TryBuildTransactionWithoutPrevTx()`